src/common/commonutils/UserUtils.c

```diff
@@ -185,6 +184,16 @@ static bool IsUserNonLogin(SimplifiedUser* user)
     return noLogin;
 }
 
+static bool IsSystemAccount(SimplifiedUser* user)
+{
+    return (user && ((user->username && (0 == strcmp(user->username, g_root))) || IsUserNonLogin(user) || (user->userId < 1000))) ? true : false;
+}
+
+static bool IsSystemGroup(SimplifiedGroup* group)
+{
+    return (group && ((group->groupName && (0 == strcmp(group->groupName, g_root))) || (group->groupId < 1000))) ? true : false;
```

Comment:
same as above, why also check the group name when the groupId is all we need?

Reply:
Same as above. Because we need to check all possible conditions that we can to catch strange cases.